tests/containers/kubectl.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test the kubectl utility
+# - List pods
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+use version_utils;
+use publiccloud::utils;
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    install_kubectl();
+    # Record kubectl version and check if the tool itself is healthy
+    record_info("kubectl", script_output("kubectl version --client --output=json"));
+
+    # Configure CSP/k3s. Only one CSP at a time is possible due to the conflicting configuration in .kube/config
+    assert_script_run('mkdir -p ~/.kube');
+    my $provider = get_var("KUBECTL_CLUSTER", "k3s");
+    # Template for the extension to CSPs
+    # Currently this test only runs with the 'k3s' provider. The following is to help us to extend the test case for each CSP.
+    #if ($provider eq "AKS") {
```

Comment:
I would remove this code, the `else` statement is not even finished `els` :)

Reply:
Ok, but the `els` is intentional, because that transforms into an `elsif` with the if from the next line :pineapple: